src/index.html

```diff
@@ -16,7 +16,11 @@
       href="./styles/index.scss"
     />
```

Comment:
SCSS files should not be linked directly in HTML. Instead, compile them to CSS and link the compiled CSS file.